schemas/Routes.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Routes",
+  "description": "a list of routes from a connector",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "source_ledger": {"$ref": "Iri.json"},
+      "destination_ledger": {"$ref": "Iri.json"},
+      "connector": {"$ref": "Iri.json"},
+      "min_message_window": {"type": "number", "minimum": 0, "description": "seconds"},
+      "source_account": {"$ref": "Iri.json"},
```

Comment:
Might be helpful to add descriptions to these properties so it's clear what they're for